task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -387,7 +394,8 @@ spec:
     resources: {}
     workingDir: $(workspaces.source.path)
   volumes:
-  - emptyDir: {}
+  - emptyDir:
+      medium: Memory
```

Comment:
Do we need to keep this change and the above made to `resources`?

Reply:
I don't see a reason why we wouldn't. The buildah task itself was changed 3 days ago with this commit - https://github.com/redhat-appstudio/build-definitions/commit/f8c6a108b3be411b54075941b54142fb2b49fa5c

and the buildah-task is generated from the buildah task, so the changes are valid from my POV

Reply:
Could  you please make these changes in a separate commit? The changes look like a sync update with https://github.com/redhat-appstudio/build-definitions/commit/f8c6a108b3be411b54075941b54142fb2b49fa5c for buildah-remote. But, if it is related to the source directory change, please describe it in the commit message.

Reply:
I am working on a way to make PRs fail if they are not in sync.

Reply:
https://github.com/redhat-appstudio/build-definitions/pull/614

Reply:
That PR is now passing, once it is merged you should be able to rebase to remove the unrelated changes.

Reply:
@tkdchen Updated, removed all other changes - Stuart's #614 will take care of it.